dbms/src/Storages/DeltaMerge/File/ColumnCache.cpp

```diff
@@ -60,6 +60,50 @@ RangeWithStrategys ColumnCache::getReadStrategy(size_t pack_id, size_t pack_coun
     return range_and_strategys;
 }
 
+RangeWithStrategys ColumnCache::getReadStrategy(
+    size_t start_pack_id,
+    size_t pack_count,
+    std::unordered_set<size_t> memory_pack_ids)
+{
+    PackRange target_range{start_pack_id, start_pack_id + pack_count};
+
+    RangeWithStrategys range_and_strategys;
```

Comment:
Strategies

Reply:
keep this, it means multiple `RangeWithStrategy`

Reply:
I don't think it is a good idea, `Poco` is an example.
![image](https://github.com/pingcap/tiflash/assets/5670241/7c4ec6d6-8dbe-4476-b7ea-b3f4173a2dc5)

However, I think it isn't a big matter, don't take this as a serious review. Just my point of view.